src/e2e-test/regression-test/util/utils.spec.js

```diff
@@ -145,33 +143,33 @@ describe('Regression test testUtils - compareContent', () => {
     });
     it ('should compare correctly when given parameters that only different with fields to be deleted', () => {
         const left = testData(), right = testData();
-        right.fhirResource.entry[0].resource.date = 'different';
+        right.entry[0].resource.date = 'different';
         assert.ok(utils.compareContent(...[left, right].map(JSON.stringify)));
     });
     it ('should throw error when given unequal plain object', () => {
         const left = testData(), right = testData();
-        right.fhirResource.entry[0].resource.fullUrl = 'Differen-Url';
+        right.entry[0].resource.fullUrl = 'Differen-Url';
```

Comment:
typo Different

Reply:
Since the previous ground truths contain ui-related fields, there will be an extra `fhirResource`, currently we need not to do so because there are no ui-related fields and they will be test in other scenarioes.